selfdrive/ui/qt/onroad/annotated_camera.h

```diff
@@ -12,9 +11,11 @@
 #include "selfdrive/ui/sunnypilot/qt/onroad/model.h"
 #define ExperimentalButton ExperimentalButtonSP
 #define ModelRenderer ModelRendererSP
+#define HudRenderer HudRendererSP
 #else
 #include "selfdrive/ui/qt/onroad/buttons.h"
 #include "selfdrive/ui/qt/onroad/hud.h"
+#include "selfdrive/ui/qt/onroad/model.h"
```

Comment:
why moving this? everything works fine with it above and the HudRendererSP is called...